MoneySplitter.Models/Services/QueryParameterModel.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace MoneySplitter.Models.Services
+{
+    public class QueryParameterModel
```

Comment:
Rename properties to ParameterName, ParameterValue

Reply:
rename